TinyOS/Assembler1.s

```diff
@@ -64,7 +64,7 @@ Mega328P_Init:
 		sts	ADCSRA,r16		//stores 0x87 into ADC configuration SRAM  location
 		ldi r16,0x40		//Loads 0x40 into r16
 		sts ADMUX,r16		//Stores the value of r16 into the ADC multiplexer selection memory space in SRAM
-		ldi r16,0			//Loads 0 into r16
+		ldi r16,7			//Loads 0 into r16
```

Comment:
Can I get an explanation of this value?

Reply:
Loading the proper flags into the ADCSRB for FreeRunning Mode